process.c

```diff
@@ -8669,6 +8644,172 @@ proc_warmup(VALUE _)
  * \Module +Process+ represents a process in the underlying operating system.
  * Its methods support management of the current process and its child processes.
  *
+ * == \Process Creation
+ *
+ * Each of these methods creates a process:
+ *
+ * - Process.exec: Replaces the current process by running a given external command.
+ * - Process.spawn: Creates a child process.
+ * - Kernel#spawn: Executes the given command and returns its pid without waiting for completion.
+ * - Kernel#system: Executes the given command in a subshell.
+ *
+ * Each of these methods accepts:
+ *
+ * - An optional hash of environment variable names and values.
+ * - An optional hash of execution options.
+ *
+ * === Execution Environment
+ *
+ * Optional leading argument +env+ is a hash of name/value pairs,
```

Comment:
```suggestion
 * Optional leading argument _env_ is a hash of name/value pairs,
```

Reply:
@nobu, there are thousands of existing argument mentions using plus-signs.  Maybe I should add something to the documentation guide about that; also, when to use italics for something like _n_?

Reply:
I too often overlook inconsistencies in the notation.
I agree that the criteria should be clear.

Reply:
Is there a reason we use italics over monofont text for variable names? I took a look at the current documentation and it looks like most of it uses monofont instead of italics. I personally prefer monofont over italics because I think that monofont is for code while italics is used for emphasis.

Reply:
Just originally RDoc used `_var_`.
Also I don't necessarily want it to be rendered in italics, and monotype (with other attributes) would be fine.
I just focus on the difference between variables and other arbitrary code notations.

Reply:
Stet, per our discussion at the documentation guide.